src/aks-preview/azext_aks_preview/azurecontainerstorage/_helpers.py

```diff
@@ -63,7 +63,7 @@ def perform_role_operations_on_managed_rg(
     assign
 ):
     managed_rg_role_scope = build_role_scope(node_resource_group, None, subscription_id)
-    roles = ["Azure Container Storage Operator", "Reader"]
+    roles = ["Azure Container Storage Operator"]

```

Comment:
May I ask will this change cause breaking changes?

Reply:
This won't be a breaking change as the reader role is not necessary.